gis/gccview/pull_interventions.py

```diff
@@ -77,7 +77,10 @@ def send_tempdata(output_table, insert_column, return_json):
     fields = return_json['fields']
     trials = [[field['name'], field['type']] for field in fields]
     for feature in features:
-        geom = feature['geometry']
+        try:
+            geom = feature['geometry']
```

Comment:
`geom = feature.get('geometry')` is a simpler solution

Reply:
changed to get